apps/tasks/lib/components/task_status_pill_box.dart

```diff
@@ -40,7 +40,6 @@ class TaskStatusPillBox extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    Color backgroundColor = dividerColor ?? Colors.white;
```

Comment:
Provide `Colors.transparent` as default value for `dividerColor`.

Reply:
There is no default value here anymore, instead the default is given by the underlying `TaskStatusPill`, do you want to also set a default in this class?

Reply:
The real question is do we need a background color?
We could use transparency directly?! I don't see any szenario where we have to set the background color manually? Do you see one?

Reply:
When you only want to color the diagonals, then the background color is relevant. To change the diagonal color you would need to add a colored container in the background, which has to match the width and the border radius of the pillbox, which would be require much effort, therefore the attribute.
But we don't use it yet. At the current state it behaves as you want it to by being transparent and therefore I do not see a reason to remove this additional attribute

Reply:
Allright. For the next time we could save the more effort. I am quite sure that we will never touch this attribute.